eng/Subsets.props

```
@@ -587,6 +587,7 @@
       <AdditionalProperties Condition="'%(ProjectToBuild.Category)' == 'tools'">%(AdditionalProperties);Configuration=$(ToolsConfiguration)</AdditionalProperties>
 
       <!-- Propagate configurations for cross-subset builds -->
+      <AdditionalProperties>%(AdditionalProperties);CoreCLRConfiguration=$(CoreCLRConfiguration)</AdditionalProperties>
```

Does this work if we propagate the `RuntimeConfiguration` instead? I'd rather avoid hard-coding a specific runtime here  if possible.

```suggestion
      <AdditionalProperties>%(AdditionalProperties);RuntimeConfiguration=$(RuntimeConfiguration)</AdditionalProperties>
```